src/App.js

```diff
@@ -7,16 +7,21 @@ import Books from './components/Books';
 import InputForm from './components/InputForm';
 import './components/styles/Navbar.css';
 import Categories from './components/Categories';
+import './components/styles/Books-section.css';
+import avatar from './components/asset/avatar.png';
 
 function App() {
   return (
     <BrowserRouter>
-      <div>
-        <h1>Bookstore CMS</h1>
-        <ul className="form">
-          <li><Link to="/">Books</Link></li>
-          <li><Link to="/Categories">Categories</Link></li>
-        </ul>
+      <div className="body">
+        <div className="header">
+          <ul className="Nav-bar">
+            <li className="Project-header">Bookstore CMS</li>
+            <li><Link className="Books" to="/">Books</Link></li>
+            <li><Link className="categories" to="/Categories">CATEGORIES</Link></li>
+          </ul>
+          <img alt="logo" className="logo" src={avatar} />
+        </div>
```

Comment:
You did a great job working on this milestone, however, there's still room to improve: One of the main goals of this project is to follow as closely as possible a given design, this includes color, fonts, and layout. 🚀 

|Template|
|---|
|![image](https://user-images.githubusercontent.com/77069011/233786336-f08823ed-17df-49bb-8de9-8383ebd159dd.png)|

|Your design|
|---| 
![image](https://user-images.githubusercontent.com/77069011/233786274-7fe30118-dea8-4351-9ba7-5676887956ea.png) |

- [ ] Please consider using the correct font in your design! 🤓 🔧 
- [ ] Please also consider working on the styles for the form inputs, they're different than the given design! 🤓 🔧